src/components/ContentEditor/WidgetEditor/WidgetEditor.tsx

```diff
@@ -59,6 +60,12 @@ export const WidgetEditor: React.FC<WidgetEditorProps> = ({ content, contentHash
             <div>
                 <Button variant="contained" color="primary"
```

Comment:
You might want to consider making this button disabled and adding in a "force edit anyways" button instead.